### ISSUE TITLE
No live preview until file is saved.

### ISSUE THREAD

**Issue by jremydeaton** [jasonsanjose/brackets-sass#85]
While this is not really a huge deal, It doesn't work as demonstrated in the video. Making changes to my main "style.scss" or included paths files, I get no live preview unless I save. Tried with my own project as well as the example project. This is the same for both Mac and Windows. Using the Brackets default live preview. 

Another question... Not sure is this is a factor but my file structure is this:
.brackets.json
app/
-- styles/
---- style.scss
---- base/
---- modules/
---- components/

Do I need to set the include path for the directories under the styles directory, and if so, should it be:
"./" or "app/styles/"

**Comment by jremydeaton** [comment]
Found out what the issue was. The line endings changed in the .brackets.json file when I pushed it up to my git repository, which caused this problem. I recreated the file from scratch and had no issues....
That is until I pushed the project up to my repository (at work from Windows machine). When I got home and pulled it on my Mac, I encountered a new error: It KILLED the live preview completely.

To remedy this, I had to gitignore the .brackets.json and recreate a duplicate for each machine. Annoying but the issue is fixed.

As this no longer seems to an issue with your Brackets SCSS, I am voluntarily closing this issue and opening up with Brackets itself.

Thanks.
